umake/frameworks/ide.py

```diff
@@ -93,9 +93,9 @@ def parse_download_link(self, line, in_download):
         else:
             in_download = False
         if in_download:
-            p = re.search(r"href='(http://www\.eclipse\.org\/downloads/download\.php\?file=.*\.tar\.gz)'", line)
+            p = re.search(r"href='(//www\.eclipse\.org\/downloads/download\.php\?file=.*\.tar\.gz)'", line)
```

Comment:
```suggestion
            p = re.search(r"href='(http:|https:)?(//www\.eclipse\.org\/downloads/download\.php\?file=.*\.tar\.gz)'", line)
```

I'd change the regex so it works with and without the https prefix.
The checksum pages seem to all be available via https, so we don't need to check that.